### DISCUSSION TITLE
Add a link to GitHub repo on the published website

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#16] [General (Open Discussions)]
**Moved from: https://github.com/canada-ca/a11y/issues/5**

To encourage contribution (and reduce confusion over who is responsible for the content), I suggest we add a link to this repo and make it clear that this is an interdepartmental collaborative effort and contribution is welcome.

cc/ @juleskuehn

**Comment by shawnthompson** [comment]
Plan on making a contribute section above the footer.

**Reply by delisma** [reply]
TIL that the new footer coming on Canada.ca will have a contextual band (1) that could be used to add contextual contact links like GitHub links. It won't be as prominent as the one that you're proposing but you could **campaign** to allow it to have more flexibility..

![](https://test.canada.ca/experimental/design-system/images/footer-en.png)

**Reply by shawnthompson** [reply]
And how does one **campaign** for something like this?

**Reply by delisma** [reply]
With DTO and PP. I think in your case you could start by asking how we can add icons next to links of third-party sites. Then I don't know yet how you would add the Netlify link (if it's really necessary).

**Reply by shawnthompson** [reply]
The Netlify link is necessary because we are on the [Open Source Plan Policy](https://www.netlify.com/legal/open-source-policy/) and it's part of the agreement. We might be able to only show it on the development servers because the live server is running off Github Pages.
